web/templates/page/invitation.html.eex

```diff
@@ -0,0 +1,47 @@
+<%= if logged_in?(@conn) do %>
+  <fieldset>
+    <legend>Request Invitation</legend>
+    <p>
+      <label>Invite a Friend...</label>
+    </p>
+    <p>
+      <label for="email" class="inputLabel">Email:</label>
+      <input type="text" class="inputField" id="email" name="email" value="" size="20" maxlength="50" />
+    </p>
+    <input type="submit" class="inputButton" id="submit" name="submit" value="Request key" />
+  </fieldset>
+  <script>
+    $('#submit').click(function() {
+      email = $('#email').val();
+
+      $.ajax("<%= account_path(@conn, :request_invite) %>", {
+        type: 'POST',
+        contentType: 'application/json',
+        dataType: 'json',
+        data: JSON.stringify({"email": email}),
+        cache: false
+      })
+      .done(function(data) {
+
+        if(data.status == "error"){
+          $('.authMessage').append("<br/> " + data.message);
+        }
+        else {
+          $('.authMessage').append("<br/> Invitation is created.");
+          $('.authMessage').append("<br/> Email: " + data.email);
+          $('.authMessage').append("<br/> InviteKey: " + data.key);
+          $('.authMessage').append("<p>Send as Email<a href=\ mailto:").append(email).append("?Subject=Hello%20again target=\"_top\">Send Mail</a></p>");
```

Comment:
If we have this link anyway, i think we can also add a short body with the data.key, no? Otherwise the user can simply create the email herself... ;)